src/lib/utils/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {
+  Fade,
+  makeStyles,
+  Tooltip as MuiTooltip,
+  TooltipProps as MuiTooltipProps,
+} from '@material-ui/core'
+
+export type TooltipProps = MuiTooltipProps
+
+const useTooltipStyle = makeStyles(theme => ({
+  arrow: {
+    // gray-700
+    color: '#878E95',
+  },
+  tooltip: {
+    // gray-700
+    backgroundColor: '#878E95',
+    fontSize: '14px',
+    borderRadius: '2px',
+  },
+}))
```

Comment:
Styles pulled from Figma. Think we should wait to fix the references to use variables rather than hard-coded colors when we go to MUI v5